r-p-s.js

```diff
@@ -1,83 +1,64 @@
 const drawsSpan = document.querySelector('#draws');
 const percentSpan = document.querySelector('#percent');
+const resetButton = document.querySelector('#reset-button');
+
 
-// initialize state
 let wins = 0;
 let losses = 0;
 let draws = 0;
 let total = 0;
 
+
 // set event listeners to update state and DOM
 
 button.addEventListener('click', () => {
     
     const checkedButton = document.querySelector(':checked');
     const userInput = checkedButton.value;
     const computerInput = getRandomThrow();
-
-    if (userInput === 'rock' && computerInput === 'paper') {
+    const result = checkResults(userInput, computerInput);
+    if (result === 'win') {
+        wins++;
+        resultsSpan.textContent = 'You won!';
+    } else if (result === 'lose') {
         losses++;
-        total++;
+        resultsSpan.textContent = 'You lose!';
 
-        resultsSpan.textContent = 'Paper covers rock, you lose!';
-        lossesSpan.textContent = losses;
-        percentSpan.textContent = `${(wins / total) * 100}%`;
-    
-    } else if (userInput === 'paper' && computerInput === 'scissors') {
-        losses++;
-        total++;
+    } else {
+        draws++;
+        resultsSpan.textContent = 'Its a draw!';
 
-        resultsSpan.textContent = 'Scissors cuts paper, you lose!';
-        lossesSpan.textContent = losses;
-        percentSpan.textContent = `${(wins / total) * 100}%`;
-    
-    } else if (userInput === 'scissors' && computerInput === 'rock') {
-        losses++;
-        total++;
+    }
 
-        resultsSpan.textContent = 'Rock smashes scissors, you lose!';
-        lossesSpan.textContent = losses;
-        percentSpan.textContent = `${(wins / total) * 100}%`;
-    
-    } else if (userInput === 'rock' && computerInput === 'scissors') {
-        wins++;
-        total++;
+    winsSpan.textContent = wins;
+    lossesSpan.textContent = losses;
+    drawsSpan.textContent = draws;
 
-        resultsSpan.textContent = 'Rock smashes scissors, you win!';
-        winsSpan.textContent = wins;
-        percentSpan.textContent = `${(wins / total) * 100}%`;
-    
-    } else if (userInput === 'paper' && computerInput === 'rock') {
-        wins++;
-        total++;
+    total++;
+    percentSpan.textContent = `${((wins / total) * 100).toFixed(1)}%`;
 
-        resultsSpan.textContent = 'Paper covers rock, you win!';
-        winsSpan.textContent = wins;
-        percentSpan.textContent = `${(wins / total) * 100}%`;
-    
-    } else if (userInput === 'scissors' && computerInput === 'paper') {
-        wins++;
-        total++;
+});
 
-        resultsSpan.textContent = 'Scissors cut paper, you win!';
-        winsSpan.textContent = wins;
-        percentSpan.textContent = `${(wins / total) * 100}%`;
-    
-    } else {
-        draws++;
-        total++;
+resetButton.addEventListener("click", () => {
+    wins = 0;
+    losses = 0;
+    draws = 0;
+    total = 0;
+    percentSpan.textContent = "";
+    winsSpan.textContent = wins;
```

Comment:
Will need to have single quotes '' in your files, Github is failing due to these checks